### PR TITLE
update polkadot, substrate, cumulus readme

### PR DESCRIPTION
**Pull request by the-right-joyce**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Welcome to our new repository! As long anticipated, we have finally merged all three repositories (Cumulus, Polkadot and Substrate) into a single unified entity, marking a significant milestone in our journey.
```
Comment (bkchr) [inline README.md:1]: IMO, we should write a proper `README`. This means no welcoming and that this is a new repo etc.

Comment (the-right-joyce) [reply README.md:1]: okay.

**Review by gilescope — APPROVED** [review]

**Review by ggwpez — APPROVED** [review]

**Review by liamaharon — CHANGES_REQUESTED** [review]
There're a few `paritytech/substrate`, `paritytech/cumulus`, `paritytech/polkadot` urls that need to up updated to `paritytech/polkadot-sdk`

**Review by mrcnski — APPROVED** [review]

**Comment by the-right-joyce** [comment]
@liamaharon updated, please check

File: cumulus/README.md [hunk 2]
```diff
@@ -1,13 +1,265 @@
...
+zombienet --provider native spawn ./zombienet/examples/small_network.toml
+```
+
+### Manual Setup
+#### Launch the Relay Chain
+
+```bash
+# Clone
+git clone https://github.com/paritytech/polkadot-sdk
+cd polkadot-sdk/polkadot
```
Comment (liamaharon) [inline cumulus/README.md:149]: I don't think this `cd` should be here. `cargo b ...` works fine in the root, and also the `target` lives in the root so this breaks the following commands prefixed with `./target` 



```suggestion
```

File: cumulus/README.md [hunk 3]
```diff
@@ -1,13 +1,265 @@
...
+# Bob (In a separate terminal)
+./target/release/polkadot --chain rococo-local-cfde.json --bob --tmp --port 30334
+```
+
+#### Launch the Parachain
+
+```bash
+# Clone
+git clone https://github.com/paritytech/polkadot-sdk
+cd polkadot-sdk/cumulus
```
Comment (liamaharon) [inline cumulus/README.md:169]:
```suggestion
```

File: polkadot/README.md [hunk 4]
```diff
@@ -1,13 +1,264 @@
...
+
+```bash
+cargo test --workspace --release
+```
+
+You can start a development chain with:
+
+```bash
+cargo build
+cargo run -- --dev
```
Comment (liamaharon) [inline polkadot/README.md:201]:
```suggestion
cargo run --bin polkadot -- --dev
```

File: polkadot/README.md [hunk 5]
```diff
@@ -1,13 +1,264 @@
...
+```
+
+Detailed logs may be shown by running the node with the following environment variables set:
+
+```bash
+RUST_LOG=debug RUST_BACKTRACE=1 cargo run -- --dev
```
Comment (liamaharon) [inline polkadot/README.md:207]:
```suggestion
RUST_LOG=debug RUST_BACKTRACE=1 cargo run --bin polkadot -- --dev
```

File: polkadot/README.md [hunk 6]
```diff
@@ -1,13 +1,264 @@
...
+```
+
+### Development
+
+You can run a simple single-node development "network" on your machine by running:
+
+```bash
+polkadot --dev
```
Comment (liamaharon) [inline polkadot/README.md:215]: there're no instructions to build the polkadot binary and put it on your path in this section

```suggestion
cargo run --bin polkadot --release -- --dev
```

File: polkadot/README.md [hunk 7]
```diff
@@ -1,13 +1,264 @@
...
+You can muck around by heading to <https://polkadot.js.org/apps> and choose "Local Node" from the
+Settings menu.
+
+### Local Two-node Testnet
+
+If you want to see the multi-node consensus algorithm in action locally, then you can create a
+local testnet. You'll need two terminals open. In one, run:
+
+```bash
+polkadot --chain=polkadot-local --alice -d /tmp/alice
```
Comment (liamaharon) [inline polkadot/README.md:227]:
```suggestion
cargo run --bin polkadot --release -- --chain=polkadot-local --alice -d /tmp/alice
```

File: polkadot/README.md [hunk 8]
```diff
@@ -1,13 +1,264 @@
...
+```
+
+And in the other, run:
+
+```bash
+polkadot --chain=polkadot-local --bob -d /tmp/bob --port 30334 --bootnodes '/ip4/127.0.0.1/tcp/30333/p2p/ALICE_BOOTNODE_ID_HERE'
```
Comment (liamaharon) [inline polkadot/README.md:233]:
```suggestion
cargo run --bin polkadot --release -- --chain=polkadot-local --bob -d /tmp/bob --port 30334 --bootnodes '/ip4/127.0.0.1/tcp/30333/p2p/ALICE_BOOTNODE_ID_HERE'
```

File: polkadot/README.md [hunk 9]
```diff
@@ -1,13 +1,264 @@
...
+
+### Shell Completion
+
+[Shell Completion](doc/shell-completion.md)
+
+## Contributing
+
+### Contributing Guidelines
+
+[Contribution Guidelines](https://github.com/paritytech/polkadot-sdk/blob/master/docs/CONTRIBUTING.md)
```
Comment (liamaharon) [inline polkadot/README.md:256]: broken link

```suggestion
[Contribution Guidelines](https://github.com/paritytech/polkadot-sdk/blob/master/polkadot/CONTRIBUTING.md)
```

File: polkadot/README.md [hunk 10]
```diff
@@ -1,13 +1,264 @@
...
+
+### Contributor Code of Conduct
+
+[Code of Conduct](https://github.com/paritytech/polkadot-sdk/blob/master/docs/CODE_OF_CONDUCT.md)
```
Comment (liamaharon) [inline polkadot/README.md:260]: broken link

```suggestion
[Code of Conduct](https://github.com/paritytech/polkadot-sdk/blob/master/polkadot/CODE_OF_CONDUCT.md)
```

File: substrate/README.md [hunk 11]
```diff
@@ -1,13 +1,35 @@
-Dear contributors and users,
+# Substrate &middot; [![GitHub license](https://img.shields.io/badge/license-GPL3%2FApache2-blue)](#LICENSE) [![GitLab Status](https://gitlab.parity.io/parity/substrate/badges/master/pipeline.svg)](https://gitlab.parity.io/parity/substrate/pipelines) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](docs/CONTRIBUTING.md) [![Stack Exchange](https://img.shields.io/badge/Substrate-Community%20&%20Support-24CC85?logo=stackexchange)](https://substrate.stackexchange.com/)
```
Comment (liamaharon) [inline substrate/README.md:1]:
```suggestion
# Substrate &middot; [![GitHub license](https://img.shields.io/badge/license-GPL3%2FApache2-blue)](#LICENSE) [![GitLab Status](https://gitlab.parity.io/parity/mirrors/polkadot-sdk/badges/master/pipeline.svg)](https://gitlab.parity.io/parity/mirrors/polkadot-sdk/-/pipelines) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](docs/CONTRIBUTING.md) [![Stack Exchange](https://img.shields.io/badge/Substrate-Community%20&%20Support-24CC85?logo=stackexchange)](https://substrate.stackexchange.com/)
```

File: substrate/README.md [hunk 12]
```diff
@@ -1,13 +1,35 @@
...
 
-Best Regards,
+Join the highly active and supportive community on the [Substrate Stack Exchange](https://substrate.stackexchange.com/) to ask questions about use and problems you run into using this software.
+Please do report bugs and [issues here](https://github.com/paritytech/polkadot-sdk/issues) for anything you suspect requires action in the source. 
 
-Parity Technologies
\ No newline at end of file
+## Contributions & Code of Conduct
+
+Please follow the contributions guidelines as outlined in [`docs/CONTRIBUTING.md`](https://github.com/paritytech/polkadot-sdk/blob/master/docs/CONTRIBUTING.md).
```
Comment (liamaharon) [inline substrate/README.md:21]:
```suggestion
Please follow the contributions guidelines as outlined in [`docs/CONTRIBUTING.md`](https://github.com/paritytech/polkadot-sdk/blob/master/substrate/docs/CONTRIBUTING.md).
```

File: substrate/README.md [hunk 13]
```diff
@@ -1,13 +1,35 @@
...
+In all communications and contributions, this project follows the [Contributor Covenant Code of Conduct](https://github.com/paritytech/polkadot-sdk/blob/master/docs/CODE_OF_CONDUCT.md).
```
Comment (liamaharon) [inline substrate/README.md:22]:
```suggestion
In all communications and contributions, this project follows the [Contributor Covenant Code of Conduct](https://github.com/paritytech/polkadot-sdk/blob/master/substrate/docs/CODE_OF_CONDUCT.md).
```

File: substrate/README.md [hunk 14]
```diff
@@ -1,13 +1,35 @@
...
+
+## Security
+
+The security policy and procedures can be found in [`docs/SECURITY.md`](https://github.com/paritytech/polkadot-sdk/blob/master/docs/SECURITY.md).
```
Comment (liamaharon) [inline substrate/README.md:26]:
```suggestion
The security policy and procedures can be found in [`docs/SECURITY.md`](https://github.com/paritytech/polkadot-sdk/blob/master/substrate/docs/SECURITY.md).
```

**Review by liamaharon — CHANGES_REQUESTED** [review]

File: polkadot/README.md (hunk 10 above)
Comment (the-right-joyce) [reply polkadot/README.md:260]: yes bc it's related to my other PR that needs to be merged

File: substrate/README.md (hunk 12 above)
Comment (the-right-joyce) [reply substrate/README.md:21]: same here #1181

File: substrate/README.md (hunk 13 above)
Comment (the-right-joyce) [reply substrate/README.md:22]: #1181

**Review by liamaharon — APPROVED** [review]

**Comment by davxy** [comment]
Same as https://github.com/paritytech/polkadot-sdk/pull/1181#issuecomment-1696257237

I see that these conventions here are sometimes applied and sometimes not.

l'd suggest to keep the formatting consistent following a convention

**Review by alvicsam — APPROVED** [review]

**Pull request merged**